common/lib/capa/capa/templates/status_span.html

```diff
@@ -0,0 +1,5 @@
+<%page expression_filter="h" args="status, sr_text, classname, status_id"/>
+<span class="status ${classname}" id="status_${status_id}" data-tooltip="${status.display_tooltip}">
+  <span class="sr">${sr_text}</span>
```

Comment:
How about just getting the sr_text directly from status as it is already being passed in (status.display_name)?